crypto/evp/evp_lib.c

```diff
@@ -902,12 +903,32 @@ int (*EVP_MD_meth_get_ctrl(const EVP_MD *md))(EVP_MD_CTX *ctx, int cmd,
     return md->md_ctrl;
 }
 
+#ifndef OPENSSL_NO_DEPRECATED_3_0
 const EVP_MD *EVP_MD_CTX_md(const EVP_MD_CTX *ctx)
 {
     if (ctx == NULL)
         return NULL;
     return ctx->reqdigest;
```

Comment:
You could change these lines to:

``` C
    return EVP_MD_CTX_get0_md(ctx);
```

Reply:
[just call the get0 function] I thought of that, but breaking const to add it back was a little too much for me. :)